charts/redpanda/templates/connectors/connectors.yaml

```
@@ -43,7 +43,7 @@ limitations under the License.
 {{ $command := list }}
 {{ if (include "sasl-enabled" . | fromJson).bool }}
   {{ $command = concat $command (list "sh" "-c") }}
-  {{ $consoleSASLConfig := (printf "set -e; IFS=: read -r CONNECT_SASL_USERNAME KAFKA_SASL_PASSWORD CONNECT_SASL_MECHANISM < $(find /mnt/users/* -print); CONNECT_SASL_MECHANISM=${CONNECT_SASL_MECHANISM:-%s}; export CONNECT_SASL_USERNAME CONNECT_SASL_PASSWORD_FILE CONNECT_SASL_MECHANISM;" (( include "sasl-mechanism" . ) | lower)) }}
+  {{ $consoleSASLConfig := (printf "set -e; IFS=: read -r CONNECT_SASL_USERNAME KAFKA_SASL_PASSWORD CONNECT_SASL_MECHANISM < <(grep \"\" $(find /mnt/users/* -print)); CONNECT_SASL_MECHANISM=${CONNECT_SASL_MECHANISM:-%s}; export CONNECT_SASL_USERNAME CONNECT_SASL_PASSWORD_FILE CONNECT_SASL_MECHANISM;" (( include "sasl-mechanism" . ) | lower)) }}
```

I don't know if that matters, but maybe
```suggestion
  {{ $consoleSASLConfig := (printf "set -e; IFS=':' read -r CONNECT_SASL_USERNAME KAFKA_SASL_PASSWORD CONNECT_SASL_MECHANISM < <(grep \"\" $(find /mnt/users/* -print)); CONNECT_SASL_MECHANISM=${CONNECT_SASL_MECHANISM:-%s}; export CONNECT_SASL_USERNAME CONNECT_SASL_PASSWORD_FILE CONNECT_SASL_MECHANISM;" (( include "sasl-mechanism" . ) | lower)) }}
```

---

It doesnt, but ill commit it